docs-site/package.json

```diff
@@ -14,16 +14,17 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "2.4.1",
-    "@docusaurus/preset-classic": "2.4.1",
+    "@docusaurus/core": "3.4.0",
+    "@docusaurus/preset-classic": "3.4.0",
     "@mdx-js/react": "^1.6.22",
     "clsx": "^1.2.1",
```

Comment:
**nitpick (blocking):** Can you update this to `^2.1.1`

Reply:
Updated in 276320f4653f589781ce3dfc58ec9af8a1aec1e0